requirements.txt

```diff
@@ -1,110 +1,53 @@
-amqp==2.5.2
-anyjson==0.3.3
+# narrowed down using https://github.com/bndr/pipreqs
+# following 5 were recommended by pipreqs but weren't in our pip freeze from hs container
+# gdal==3.5.2
+# osgeo==0.0.1
+# osr==0.0.1
+# Pillow==9.2.0
+# recommonmark
+# then INSTALLED_APPS were added manually
 arrow==0.7.0
-autopep8==1.2.2
-bagit==1.5.4
 celery==4.4.0
-cffi==1.11.5
-click==6.7
-coreapi==2.3.3
-coverage==4.0.3
-cycler==0.10.0
-#defusedexpat==0.4
-defusedxml==0.4.1
-Django==1.11.29
-django-appconf==1.0.2
-django-autocomplete-light==2.3.3
-django-braces==1.9.0
-django-compressor==2.2
-django-contrib-comments==1.9.0
-django-cors-headers==2.4.0
-django-crispy-forms==1.7.2
-django-debug-toolbar==1.9.1
-django-freshly==0.1.2
-django-haystack==2.8.1
+deepdiff==1.7.0
+defusedexpat #included in dockerfile separately
+Django==3.2.15
+django-autocomplete-light==2.3.6
+django-contrib-comments==2.2.0
+django-cors-headers==3.10.1
+django-crispy-forms==1.13.0
+django-haystack==3.1.1
 django-heartbeat==2.0.2
-django-ipware==2.1.0
-django-jsonfield==1.0.1
-django-modeltranslation==0.12.2
 django-nose==1.4.5
-django-oauth-toolkit==1.1.0
-django-robots==2.0
-django-security==0.9.3
+django-oauth-toolkit==2.1.0
+django-robots==4.0
+django-security==0.12.0
 django-test-without-migrations==0.6
-django-timedeltafield==0.7.10
 django-widget-tweaks==1.4.1
-djangorestframework==3.11.0
-docker-py==1.7.2
+djangorestframework==3.13.1
 dominate==2.4.0
-DoubleMetaphone==0.1
-drf-haystack==1.8.5
-drf-yasg==1.17.1
-ecdsa==0.13.3
-enum34==1.1.6
-filebrowser-safe==0.5.0
-flake8==3.5.0
-flex==6.14.1
-flower==0.9.3
-funcsigs==1.0.2
-#functools32==3.2.3.post2
+drf-yasg==1.20.0
 future==0.16.0
-geojson==1.3.2
-geopy==1.16.0
-gevent==1.4.0
-grappelli-safe==0.5.1
-gunicorn==19.6.0
-html5lib==0.9999999
-inflection==0.3.1
-isodate==0.5.4
-Jinja2==2.10.1
-jsonpointer==1.14
+django-heartbeat==2.0.2
+hsmodels==0.5.3
 jsonschema==2.6.0
-keepalive==0.5
 kombu==4.6.7
 lxml==4.4.0
-#mapnik==0.1
 Markdown==3.0.1
-MarkupSafe==1.0
-matplotlib==3.1.2
-mccabe==0.6.1
-Mezzanine==4.3.1
-mock==1.3.0
+Mezzanine==5.1.4
 nameparser==0.5.7
-netCDF4==1.5.1.2 
-nose==1.3.7
-nose-timer>=0.7.4,<1.0.0
-oauthlib==3.1.0
-OWSLib==0.10.3
+netCDF4==1.5.1.2
+numpy==1.16.* # also included in dockerfile (order matters)
 paramiko==2.4.2
-pbr==1.10.0
-pep8==1.7.0
-prettytable==0.7.2
 probablepeople==0.5.4
-psycopg2==2.8.2
-ipaddress==1.0.22
-pyasn1==0.4.4
-pycodestyle==2.3.1
 PyCRS==1.0.1
-pyflakes==1.6.0
-pylint==1.5.5
-pyproj==1.9.5.1
-pysolr==3.4.0
-pysqlite3==0.2.1
-pytest-django>=3.4.7, <4.0.0
-python-crfsuite==0.9.6
-python-dateutil==2.5.3
-python-irodsclient==0.8.1 
-pytz==2016.3
-PyYAML==5.3
-rcssmin==1.0.6
-rdflib==4.0.1
-redis==2.10.5
-requests==2.20.0
-rfc3987==1.3.8
+pydantic # pydantic seemed to be missing
+pyproj
+python-dateutil==2.8.*
+pytz==2021.*
+rdflib==5.0.0
+requests==2.27.1
 robot-detection==0.4
-selenium==3.0.2
```

Comment:
requirements.txt in this PR has all of the direct dependencies of HS django app.
But when I see that the old version had things like "selenium" ... I wonder -> do we have additional python requirements for services external to the django app? If so, I can easily add these back in, I just wanted to avoid dependency bloat if possible, so only included those that I could see a direct need for.